steps/send-google-chat-message/0.1.0/step.yml

```diff
@@ -0,0 +1,57 @@
+title: Send Google Chat message
+summary: |
+  Send a Google Chat message to a private or group channel via Bot!
+description: |
+  Send a Google Chat message to a private or group channel via Bot!
+website: https://github.com/kyuuuyki/bitrise-step-send-google-chat-message
+source_code_url: https://github.com/kyuuuyki/bitrise-step-send-google-chat-message
+support_url: https://github.com/kyuuuyki/bitrise-step-send-google-chat-message/issues
+published_at: 2020-11-13T16:54:21.475976+07:00
+source:
+  git: https://github.com/kyuuuyki/bitrise-step-send-google-chat-message.git
+  commit: 0bb7a589cf782d7e7247d67dc7896081525acd05
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- GOOGLE_CHAT_WEBHOOK_URL: Required
+  opts:
+    description: |
+      See https://developers.google.com/hangouts/chat/how-tos/webhooks
+    is_expand: true
+    is_required: true
+    summary: Google Chat incoming webhooks link.
+    title: Google Chat Webhook URL
+    value_options: []
+- JIRA_ORGANIZATION_NAME: Optional
```

Comment:
What is the relationship between Jira and Google hangouts?
It should be explained in description.